packages/@glimmer/addon-blueprint/files/testem.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Lets use `testem.js` instead

Reply:
Hmm, the files from here down _basically_ see like the app files without lots of modifications other than the paths. Can we avoid duplicating them like this?

ember-cli itself avoids duplicating by using a `fileMap` to move the app files into `tests/dummy`

https://github.com/ember-cli/ember-cli/blob/6e84ea4da023960f89404a781375223e6c833609/blueprints/addon/index.js#L151-L176